### PR TITLE
chore(scalars): adjust ToggleField stories to new variant proposal

### PR DESCRIPTION
**Pull request by alejandrocabriales**
## Ticket
https://trello.com/c/5oApGmej/761-standardize-storybook-documentation-for-component-consistency

## Description
Adjust ToggleField stories to new variant proposal

### CODE REVIEW

File: packages/design-system/src/scalars/components/fragments/toggle-field/toggle.stories.tsx [hunk 1]
```diff
@@ -1,64 +1,30 @@
...
-    },
-    onChange: {
-      action: "checked changed",
-      description: "Callback when the checked state changes",
-      table: {
-        type: { summary: "(checked: boolean) => void" },
+    ...getDefaultArgTypes({
+      enabledArgTypes: {
+        value: true,
+        defaultValue: true,
```
Comment (YasielCabrera) [inline packages/design-system/src/scalars/components/fragments/toggle-field/toggle.stories.tsx]: value and default value are enabled by default. But you should add `valueControlType: "boolean"` and `valueType: "boolean"`

File: packages/design-system/src/scalars/components/fragments/toggle-field/toggle.stories.tsx [hunk 2]
```diff
@@ -1,64 +1,30 @@
...
-      description: "Name attribute for the toggle input",
-    },
+      valueControlType: "boolean",
+      valueType: "true",
+    }),
+    ...getValidationArgTypes({
+      enabledArgTypes: {
+        customValidator: false,
+      },
+    }),
```
Comment (YasielCabrera) [inline packages/design-system/src/scalars/components/fragments/toggle-field/toggle.stories.tsx]: the toggle field doesn't have support of `customValidator`? If now, you should add it, it is a requirement

**Review by YasielCabrera — CHANGES_REQUESTED** [review]

File: packages/design-system/src/scalars/components/fragments/toggle-field/toggle.stories.tsx (hunk 1 above)
Comment (alejandrocabriales) [reply packages/design-system/src/scalars/components/fragments/toggle-field/toggle.stories.tsx]: you are right, done

**Review by YasielCabrera — APPROVED** [review]
lgtm

**Pull request merged**